docs/customization/model.rst

```diff
@@ -96,6 +96,139 @@ That in Sylius-Standard configuration is overriden already.
                 classes:
                     model: App\Entity\Addressing\Country
 
+You can check if the configuration in ``config/packages/_sylius.yaml`` is correct by running:
+
+.. code-block:: bash
+
+    $ php bin/console debug:container --parameter=sylius.model.country.class
+
+
+If all is well the output should look like:
+
+.. code-block:: bash
+
+    ---------------------------- -------------------------------------------
+     Parameter                    Value
+    ---------------------------- -------------------------------------------
+     sylius.model.country.class   App\Entity\Country
+    ---------------------------- -------------------------------------------
+
+
+In some cases however, you will see an error:
```

Comment:
I would vote to add something like this to separate header.

Reply:
I was absolutely thinking the same thing, but what to call this?
It seems very related to "extended" models (https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/inheritance-mapping.html#class-table-inheritance ?), but how can we explain or easily find out if that is the case for the model that the developer wants to customize?

Another option could be to add a "Troubleshooting" section for example, would you be okay with that?

Reply:
`Troubleshooting` seems reasonable for me, but this is a @CoderMaggie call :)